### PR TITLE
Making the timelines aspect more prominent

### PR DESCRIPTION
**Pull request by ashishnirvana1**
When I revisited the documentation it was not obvious to me that this document covers the timelines for migrating to MV3. Hence proposing a change in title and description.

**Rendered page for review:**
https://review.docs.microsoft.com/en-us/microsoft-edge/extensions-chromium/developer-guide/manifest-v3?branch=pr-en-us-1823

### CODE REVIEW

File: microsoft-edge/extensions-chromium/developer-guide/manifest-v3.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
-title: Overview of Manifest V3
-description: Migrating from Manifest V2 to V3 will help reduce fragmentation of the web for developers and enhance privacy, security, and performance for end users.
+title: Overview and timelines for migrating to Manifest V3
```
Comment (mikehoffms) [inline microsoft-edge/extensions-chromium/developer-guide/manifest-v3.md:2]: Please change ALL places.
* "title: " yaml fileld
* h1 `#` title a few lines below it.
* toc.yml title
* full-text search this repo for the old name, update all link text.

Plan on a couple passes of this while you work with an Editor to fine-tune the new title.  It makes sense for either the PM/Dev or the Writer/Editor to make this mechanical change - there's a pretty strong argument for having Docs team handle this mechanics implementation.  @vchapel

Comment (vchapel) [reply microsoft-edge/extensions-chromium/developer-guide/manifest-v3.md:2]: resolved?

Comment (ashishnirvana1) [reply microsoft-edge/extensions-chromium/developer-guide/manifest-v3.md:2]: Thanks for reviewing and updating Victor and Mike. Is this resolved?

**Review by vchapel — APPROVED** [review]
LGTM

**Review by mikehoffms — APPROVED** [review]

File: microsoft-edge/extensions-chromium/developer-guide/manifest-v3.md (hunk 1 above)
Comment (mikehoffms) [reply microsoft-edge/extensions-chromium/developer-guide/manifest-v3.md:2]: Resolved; "title" and h2 & TOC match.

**Pull request merged**